src/main/java/com/mitchej123/hodgepodge/mixins/late/morpheus/MixinMorpheusWakePlayers.java

```diff
@@ -0,0 +1,27 @@
+package com.mitchej123.hodgepodge.mixins.late.morpheus;
+
+import net.minecraft.world.World;
+import net.minecraft.world.WorldServer;
+import net.quetzi.morpheus.SleepChecker;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(value = SleepChecker.class, remap = false)
+public class MixinMorpheusWakePlayers {
+
+    @Inject(
+            method = "advanceToMorning",
+            at = @At(
+                    value = "INVOKE",
+                    target = "Lnet/minecraft/world/WorldProvider;resetRainAndThunder()V",
+                    shift = At.Shift.BEFORE),
```

Comment:
You neee a `remap = true` here because the `remap = false` in your `@Mixin` annotations sets the default for the whole class. Otherwise this mixin will only work in a dev-environment.

Sorry, I didn't see this the last time.

Reply:
I'm not sure of the impact of this would be as i've confirmed by testing that it works in a full gtnh nightly, but i'll just change it as you said.

Reply:
Hmm, looked it up and it seems like this is a method injected by Forge, so it was correct before. Sorry for the confusion.